### PR TITLE
Move v1 package to schema package

### LINKED ISSUE #1598: Move Flux function from v1 package to schema package
Move Flux functions from v1 package to new schema package
https://github.com/influxdata/flux/pull/3228

### PR DESCRIPTION
**Pull request by pierwill**
Closes #1598.

- [x] Tests pass (no build errors)
- [x] Rebased/mergeable

### CODE REVIEW

**Comment by pierwill** [comment]
Screen Shot 2020-10-14 at 3 47 27 PM

**Comment by sanderson** [comment]
Be sure to add aliases for all the old locations.

File: content/influxdb/v2.0/reference/flux/stdlib/schema/databases.md [hunk 1]
```diff
@@ -1,21 +1,21 @@
 ---
-title: v1.databases() function
-description: The `v1.databases()` function returns a list of databases in an InfluxDB 1.7+ instance.
+title: schema.databases() function
```
Comment (nathanielc) [inline content/influxdb/v2.0/reference/flux/stdlib/schema/databases.md:2]: The databases function remained in the v1 package.

**Review by nathanielc — CHANGES_REQUESTED** [review]
LGTM with exception of the `schema.databases` function, that remains in the `v1` package. And +1 to @sanderson 's comment all these functions are still available through the `v1` package for backwards compatibility, but we should probably update those to mention they are deprecated.

Do the docs use these functions in any examples? If so I would like to see those examples updated to use the new `schema` package instead.

File: content/influxdb/v2.0/reference/flux/stdlib/schema/_index.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  influxdb_2_0_ref:
+    name: InfluxDB Schema
+    parent: Flux standard library
+weight: 202
+influxdb/v2.0/tags: [functions, influxdb-v1, package]
+---
+
+InfluxDB schema Flux functions provide tools for managing data from an InfluxDB v1.x
+database or structured using the InfluxDB v1 data structure.
+Import the `influxdata/influxdb/schema` package:
```
Comment (nathanielc) [inline content/influxdb/v2.0/reference/flux/stdlib/schema/_index.md:20]: We should remove the v1 language here, these schema functions apply equally to v1 or v2, hence why we have moved them.

File: content/influxdb/v2.0/reference/flux/stdlib/influxdb-v1/_index.md [hunk 3]
```diff
@@ -15,6 +15,11 @@ weight: 202
 influxdb/v2.0/tags: [functions, influxdb-v1, package]
 ---
 
+{{% note %}}
+Functions from the `v1` package have moved to [`schema`](/influxdb/v2.0/reference/flux/stdlib/schema).
```
Comment (nathanielc) [inline content/influxdb/v2.0/reference/flux/stdlib/influxdb-v1/_index.md:19]: Not all the functions moved, just most. Specifically the json and databases functions remain in the v1 package.

**Pull request merged**